app/app.go

```diff
@@ -84,11 +86,12 @@ import (
 	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-
-	"github.com/CosmWasm/wasmd/x/wasm"
-	wasmclient "github.com/CosmWasm/wasmd/x/wasm/client"
+	regenmodule "github.com/regen-network/regen-ledger/types/module"
 	data "github.com/regen-network/regen-ledger/x/data/module"
 	ecocredit "github.com/regen-network/regen-ledger/x/ecocredit/module"
+	group "github.com/regen-network/regen-ledger/x/group/module"
+	groupserver "github.com/regen-network/regen-ledger/x/group/server"
+	grouptypes "github.com/regen-network/regen-ledger/x/group/types"
```

Comment:
For `group` and `grouptypes` aliases, I'm just following the same format that is used for other modules in here. For `groupserver`, we could actually remove the alias.

Reply:
if you rename a directory and rename the package name to `grouptypes` you won't need to make any alias. same for `server`. Also, check`x/data` - in new modules we prefer to put types directly in the module root, so it would be `x/group`. Then you reference it as `group.MyStruct`

Reply:
Doing that was causing some import cycles back when I started working on the group module, but maybe I believe I could get around that now...